src/test/java/com/clickhouse/kafka/connect/sink/helper/SchemaTestData.java

```diff
@@ -907,6 +908,40 @@ public static Collection<SinkRecord> createZonedTimestampConversions(String topi
         return array;
     }
 
+    public static Collection<SinkRecord> createFormattedTimestampConversions(String topic, int partition) {
+        return createFormattedTimestampConversions(topic, partition, DEFAULT_TOTAL_RECORDS);
+    }
+    public static Collection<SinkRecord> createFormattedTimestampConversions(String topic, int partition, int totalRecords) {
+
+        Schema NESTED_SCHEMA = SchemaBuilder.struct()
+                .field("off16", Schema.INT16_SCHEMA)
+                .field("format_date", Schema.STRING_SCHEMA)
+                .build();
+
+
+        List<SinkRecord> array = new ArrayList<>();
+        LongStream.range(0, totalRecords).forEachOrdered(n -> {
+            Struct value_struct = new Struct(NESTED_SCHEMA)
+                    .put("off16", (short)n)
+                    .put("format_date", LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSSSSS")));
```

Comment:
Maybe we should have some more fields (year, day, ...) so we will check our conversion